docs/getting-started.md

```diff
@@ -18,7 +21,7 @@ Firecracker supports **x86_64** and **aarch64** Linux, see
 
 ### KVM
 
-Firecracker requires [the KVM Linux kernel module](https://www.linux-kvm.org/).
+Firecracker requires read/write access to the KVM module.
```

Comment:
we don't quite read/write the module itself, rather the `/dev/kvm` device.

Reply:
yeah that is what i meant, made more specific. 

> Firecracker requires read/write access to `/dev/kvm` exposed by the KVM module.